easybuild/easyblocks/h/hdf5.py

```diff
@@ -77,6 +77,9 @@ def configure_step(self):
         self.cfg.update('configopts', "--with-pic --with-pthread --enable-shared")
         self.cfg.update('configopts', "--enable-cxx --enable-fortran %s" % fcomp)
 
+        # make C API thread safe (C++ / FORTRAN APIs are unaffected)
+        self.cfg.update('configopts', "--enable-threadsafe")
+
```

Comment:
```suggestion
        self.cfg.update('configopts', "--enable-threadsafe")
        # --enable-unsupported is needed to allow --enable-threadsafe to be used together with --enable-cxx
        self.cfg.update('configopts', "--enable-unsupported")
```
and remove `--enable-unsupported` from the MPI conditional block below

Reply:
fixed in https://github.com/easybuilders/easybuild-easyblocks/pull/2824/commits/34d098c74a5e79c0f4081e88af41c9c51140269e